### PR TITLE
Shifted Setup Instructions for Telegram Bot

### LINKED ISSUE #48: Shift instruction setup of Telegram bot to README.md in that folder
Also, remove them from the main Readme file!

### PR DESCRIPTION
**Pull request by akshit-mee**
This Fixes #48. Merge and Delete Branch

### CODE REVIEW

File: Telegram/Scrapper.py [hunk 1]
```diff
@@ -38,33 +38,46 @@ def get_last_update_id(updates):
...
-        updates = get_updates(last_update_id)
-        if len(updates["result"]) > 0:
-            last_update_id = get_last_update_id(updates) + 1
-            contain_url(updates)
-        time.sleep(0.5)
 
-
-if __name__ == '__main__':
-    main()
+
```
Comment (shubhank-saxena) [inline Telegram/Scrapper.py:70]: Remove the comments if not needed

File: Telegram/Slack_Intigrater.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    metadata.append([decimal.Decimal(obj.__dict__['body']['ts']), obj.__dict__['body']['channel']])
+
+
+
+def reaction_feedback(data):
+    print(slack.reactions.get(timestamp=data[0], channel=data[1]))
+
+    
+    
+##serve_slack('Test 182')
```
Comment (shubhank-saxena) [inline Telegram/Slack_Intigrater.py:24]: Remove the comments and clean the file

File: .gitignore [hunk 3]
```diff
@@ -1,7 +1,8 @@
+  
 # Byte-compiled / optimized / DLL files
 __pycache__/
 
 node_modules
 Telegram/cred.py
 Telegram/cred.txt
-Medium/secrets/secrets.py
+Medium/secrets/secrets.py
```
Comment (shubhank-saxena) [inline .gitignore:8]: Add an extra line below

**Comment by akshit-mee** [comment]
comments needed for development in Telegram/Slack_Intigrater.py, made other changes as requested

**Pull request merged**